dev-python/parver/parver-0.1.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
```

Comment:
3.7? I guess some deps are missing it (arpeggio?); I'd appreciate if you could test them and include in this (or some other) PR if they test fine on 3.7. If they're missing tests, adding python_test would be even better.